crates/common/tedge_config/src/tedge_config_cli/models/connect_url.rs

```diff
@@ -8,6 +9,12 @@ pub struct ConnectUrl {
     host: Host,
 }
 
+impl Document for ConnectUrl {
```

Comment:
I would add a comment to explain why this cannot be derived as for `IpAddress`.

This was not obvious to me, and I had to search a bit to find this is due to the serialized type defined by `#[serde(try_from = "String", into = "String")]`.

Reply:
I've removed this now as I've found a way to avoid the impl.